LATEST_CHANGES.md

```diff
@@ -9,4 +9,8 @@
 * Add support for Mermaid diagrams (@dometto).
 * Add support for downloading page sources with ?raw (@tstein).
 * Add openssh client to docker images for ssh: repo support. (@jagerkin).
-* Add support for mathematical typesetting using KaTeX (@dometto). Users can now choose between MathJax and KaTeX with the --math flag.
\ No newline at end of file
+* Add support for mathematical typesetting using KaTeX (@dometto). Users can now choose between MathJax and KaTeX with the --math flag.
+
+## Fixes & Improvements
+
+* Fix: add git configuration for `/wiki` as safe directory. #2006
```

Comment:
Maybe indicate that this fix pertains to the Docker image specifically, e.g. with "Fix (Docker image):".

Reply:
Done! ✅